libs/sdk-common/Cargo.toml

```diff
@@ -37,4 +39,7 @@ tokio = { workspace = true }
 once_cell = { workspace = true }
 
 [build-dependencies]
-tonic-build = { workspace = true }
\ No newline at end of file
+tonic-build = { workspace = true }
+
+[features]
```

Comment:
By default all features are enabled so I suggest to change that to not include liquid. 
e.g:
`default = []`

Reply:
> By default all features are enabled so I suggest to change that to not include liquid. e.g: `default = []`

I don't know if it has changed but it's the opposite, [all features are disabled unless explicitly enabled](https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature).

Reply:
Correct, my mistake.